golismero/api/net/web_utils.py

```diff
@@ -126,7 +126,8 @@ def data_from_http_response(response):
 
     # If we have no data, return None.
     if not response.data:
-        return None
+        from ..data.information.html import HTML
```

Comment:
Sometimes server return a 404, but it not None.And None will cause an exception when add_resource() at web_utils  line 343
```python
        # Extract the data from the response.
        data = data_from_http_response(response)

        # Associate the data to the URL.
        data.add_resource(url)
```